providers/aws/resources/aws_dynamodb.go

```diff
@@ -360,14 +363,17 @@ func initAwsDynamodbGlobaltable(runtime *plugin.Runtime, args map[string]*llx.Ra
 	}
 	dynamodb := obj.(*mqlAwsDynamodb)
 
-	rawResources, err := dynamodb.globalTables()
+	rawResources := dynamodb.GetGlobalTables()
 	if err != nil {
 		return nil, nil, err
 	}
```

Comment:
This is also now checking an error that has been checked before and can be removed I think

Reply:
great catch, resolved